content/en/data_jobs/airflow.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Enable Data Jobs Monitoring for Apache Airflow
+is_beta: true
+private: true
+aliases:
+    - /data_jobs/airflow/
+    - data_jobs/airflow/
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="#" header="false" btn_hidden="true" >}}
+  Data Jobs Monitoring for Apache Airflow is in preview.
+{{< /callout >}}
+
+[Data Jobs Monitoring][1] gives visibility into the performance and reliability of workflows run by Apache Airflow DAGs.
+
+{{< tabs >}}
+{{% tab "Astronomer" %}}
+
+<div class="alert alert-warning">
+For Astronomer customers using Astro, the following setup may affect the lineage feature offered by Astro. see <a href=https://www.astronomer.io/docs/learn/airflow-openlineage#lineage-on-astro>this</a> for context.
+</div>
+
+## Requirements
+
+* [Apache Airflow 2.8.0][1] or later is required.
+* [apache-airflow-providers-openlineage][2] 1.12.0 or later is required.
+
+## Setup
+
+Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
+
+To get started, follow the instructions below.
+
+1. Ariflow provider `openlineage` is already installed in [Astro Runtimes][3]. Customize the provider version by adding the following to your `requirements.txt` file into your [Astro project][4].
+   
+   ```text
+   apache-airflow-providers-openlineage==<AIRLOW_OPENLINEAGE_PROVIDER_VERSION>
+   ```
+
+2. Configure `openlineage` provider. The simplest option is to set the following environment variables [using the Astro UI][5]:
+   
+   ```shell
+   OPENLINEAGE_URL=https://data-obs-intake.{{< region-param key="dd_site" code="true" >}}
+   OPENLINEAGE_ENDPOINT=api/v1/lineage
+   OPENLINEAGE_API_KEY=<DD_API_KEY>
+   ```
+
+   See [Astronomer official guide][5] for managing environment variables for a Deployment, and check official documentation [configuration-openlineage][6] for other supported configurations of `openlineage` provider.
+
+   **Important:**
+   * replace `<DD_API_KEY>` fully with your valid [Datadog API key][7].
+
+3. Trigger a update to your Deployment and wait for it to finish.
+
+[1]: https://github.com/apache/airflow/releases/tag/2.8.0
+[2]: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/index.html
+[3]: https://www.astronomer.io/docs/astro/runtime-provider-reference
+[4]: https://www.astronomer.io/docs/astro/cli/develop-project
+[5]: https://www.astronomer.io/docs/astro/manage-env-vars#using-the-astro-ui
+[6]: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/configurations-ref.html#configuration-openlineage
+[7]: https://docs.datadoghq.com/account_management/api-app-keys/#api-keys
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
```

Comment:
It is only for K8s, or does it also work for all self hosted installations ?

Reply:
this will work for all self-hosted installations (with some slight wording change in the tab). We can make that call out by change the tab name to `Self-hosted` if that's preferred. @warrierr , what do you think?